docs/configuration/configuration.md

```diff
@@ -0,0 +1,38 @@
+---
+sidebar_position: 0
+---
+# Cloud Service Provider Configuration
+
+Kubernetes cloud cost allocation utilizes on-demand APIs for pricing data, but additional configuration is required for cloud costs and user-specific pricing. Please refer to the specific provider pages for the cloud-specific configurations:
+
+* [AWS](aws)
+* [Azure](azure)
+* [GCP](gcp)
+* [On-Premises](on-prem)
+
+## Cloud Costs
+
+To access general cloud costs you will need to set up cost data exports for your cloud provider. Your account details and credentials will be kept in a secret file named `cloud-integration.json` with the following format only containing applicable CSPs for your installation:
+```
+{
+  "azure": [],
+  "gcp": [],
+  "aws": [],
+  "alibaba": []
+}
+```
+
+OpenCost is currently single cluster with a single provider, despite the format of the file. The content for your cloud provider's configuration are listed on their specific pages:
```

Comment:
@mattray sounds like we might want to reword this based off Webbs comment

@dwbrown2 re multi account: you would do it using this config structure

Reply:
I can remove this comment, the Allocation tab is single-cluster but Cloud Costs can be multiple cloud accounts.
https://github.com/opencost/opencost-website/pull/158#discussion_r1375111857